Makefile

```
@@ -52,6 +67,18 @@ fuzz-normalizer-test: ## Run fuzz test
 	@echo "==> Running fuzz test..."
 	$(TEST_CMD) -fuzz=Fuzz -fuzztime 50s --tags="$(UNIT_TAGS)" -race ./internal/kubernetes/operator/resources
 
+.PHONY: build-debug
+build-debug: ## Generate a binary in ./bin for debugging plugin
+	@echo "==> Building kubernetes plugin binary for debugging"
+	go build -gcflags="all=-N -l" -o ./bin/binary ./cmd/plugin
+
+.PHONY: e2e-test
+e2e-test: build-debug ## Run E2E tests
+# the target assumes the MCLI_* environment variables are exported
+	@./scripts/atlas-binary.sh
+	@echo "==> Running E2E tests..."
+	GOCOVERDIR=$(GOCOVERDIR) go test -v -p 1 -parallel 1 -v -timeout 60m -tags="e2e" ./test/e2e... $(E2E_EXTRA_ARGS)
```

GOCOVERDIR=$(GOCOVERDIR) $(TEST_CMD) -v -p 1 -parallel $(E2E_PARALLEL) -v -timeout $(E2E_TIMEOUT) -tags="$(E2E_TAGS)" ./test/e2e... $(E2E_EXTRA_ARGS)
```suggestion
	GOCOVERDIR=$(GOCOVERDIR) go test -v -p 1 -parallel 1 -v -timeout 60m -tags="e2e" ./test/e2e... $(E2E_EXTRA_ARGS)
```

lets keep the original, we need to leverage E2E_TAGS